web/source/osk/visualKeyboard.ts

```diff
@@ -1805,11 +1834,19 @@ namespace com.keyman.osk {
       } else {
         let emSizeStr = getComputedStyle(document.body).fontSize;
         let emSize = util.getFontSizeStyle(emSizeStr).val;
+
         var emScale = 1;
         if(!this.isStatic) {
           // Reading this requires the OSK to be active, so we filter out
           // BuildVisualKeyboard calls here.
-          emScale = util.getFontSizeStyle(keyman.osk._Box).val;
+          let boxFontStyle = util.getFontSizeStyle(keyman.osk._Box);
```

Comment:
So the only additional change I'd like to see is to `util.getFontSizeStyle`:

```js
     getFontSizeStyle(e: HTMLElement|string): {val: number, absolute: boolean} {
       var val: number;
       var fs: string;
 
       if(typeof e == 'string') {
         fs = e;
       } else {
-        fs = e.style.fontSize;
+        fs = getComputedStyle(e).fontSize;
       }
...
```

That copes with the situation where no style is set on `_Box` (or other elements which `getFontSizeStyle` may be called for)